calcom/cal.com
# fix:get event-types handler fix

## What does this PR do?



Fixes # (get event-types will throw an http error of status code 404, if no events are found.)



## Type of change



- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?



-  call the api at the end point http://localhost:3002/v1/event-types?apiKey=SOME_API_KEY and verify wheather it is giving a response of status code 404 with message no event types are found.


## Mandatory Tasks

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
